packages/rum-angular/tsconfig.lib.prod.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tsconfig.lib.json",
   "angularCompilerOptions": {
-    "enableIvy": true
+    "enableIvy": false
```

Comment:
Publishing fails when ivy is set to true. The recommendation from Angular team is to publish the libraries by keeping enableIvy as `false` and Applications using the library can opt in to ivy mode by using Angular Compatibility Compiler (ngcc). More details here https://next.angular.io/guide/creating-libraries#publishing-your-library